src/hotspot/share/opto/loopTransform.cpp

```diff
@@ -1679,14 +1728,82 @@ void PhaseIdealLoop::insert_atomic_post_loop(IdealLoopTree *loop, Node_List &old
   // finds some, but we _know_ they are all useless.
   peeled_dom_test_elim(loop, old_new);
   loop->record_for_igvn();
-  C->print_method(PHASE_AFTER_ATOMIC_POST, 4, post_head);
+  C->print_method(PHASE_AFTER_VECTORIZED_DRAIN, 4, post_head);
 }
 
-//------------------------------insert_atomic_post_loop_impl-------------------------------
-// The main implementation of inserting atomic post loop after vector main loop.
-void PhaseIdealLoop::insert_atomic_post_loop_impl(IdealLoopTree* loop, Node_List& old_new,
-                                                  CountedLoopNode* main_head, CountedLoopEndNode* main_end,
-                                                  Node*& main_incr, Node* limit, CountedLoopNode*& post_head) {
+//------------------------------insert_post_loop-------------------------------
+// Insert a loop as the mode specified post the given loop passed.
+
```

Comment:
```suggestion
//
```
We like to make the commenting continuous everywhere usually.